# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavutil/moz.build
# Commit: 9b17ac4800c6
# Full Hash: 9b17ac4800c6e3d2b487f965c68b4af824ed9fd4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Directly use tx instead of the older avfft interface. r=karlt,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202156
# ==============================================================================

diff -r 2e5121a988a8 -r 9b17ac4800c6 media/ffvpx/libavutil/moz.build
--- a/media/ffvpx/libavutil/moz.build	Tue Feb 27 12:40:23 2024 +0000
+++ b/media/ffvpx/libavutil/moz.build	Tue Feb 27 12:40:23 2024 +0000
@@ -75,6 +75,10 @@
         ]
         USE_LIBS += ['mozva']
 
+EXPORTS.ffvpx = [
+    "tx.h"
+]
+
 SYMBOLS_FILE =  'avutil.symbols'
 NoVisibilityFlags()
 
